BookingsAPI/Bookings.DAL/Commands/UpdateParticipantCommand.cs

```diff
@@ -66,14 +75,26 @@ public async Task Handle(UpdateParticipantCommand command)
             {
                 throw new ParticipantNotFoundException(command.ParticipantId);
             }
+            
+            var participantLinks = await _hearingService.CreateParticipantLinks(participants, command.LinkedParticipants);
 
-            participant.UpdateParticipantDetails(command.Title, command.DisplayName, command.TelephoneNumber, command.OrganisationName);
+            foreach (var participantLink in participantLinks)
+            {
+                var interpreteeLink = new LinkedParticipant(participantLink.LinkedId, 
+                    participantLink.ParticipantId, LinkedParticipantType.Interpretee);
+
+                await _context.LinkedParticipant.AddRangeAsync(participantLink, interpreteeLink);
```

Comment:
Can you re-write this but using the `AddLinkedParticipant` method in `Participant`

Reply:
Wouldn't that do a lot of things that we don't need in this?